### ISSUE TITLE
Object polymorphism for controller

### ISSUE THREAD

**Issue by Yomguithereal** [Yomguithereal/dolman#11]

**Comment by Yomguithereal** [comment]
@dheavy, here is what I mean:

``` js
var router = dolman.router({
  '/hello/:name': function(req, res) {
    return (...);
  },

  'POST /hello/:name': {
    validate: {...},
    action: function(req, res) {
      return (...);
    }
  }
});
```

**Comment by dheavy** [comment]
+1
Beautifully fluent. You did implement it as such in TOFLIT18 right?

**Comment by Yomguithereal** [comment]
Nope. Toflit uses the standard way. I can add this but I need some time because there is some parsing involved.

**Comment by Yomguithereal** [comment]
Still, there is a problem because this relies on key ordering and normally one shouldnt consider objects keys as ordered.

**Comment by dheavy** [comment]
Then let's forget it for now... It'd make the code a little less readable for a little API convenience... I don't know if it's worth it. What's your opinion?

**Comment by Yomguithereal** [comment]
I don't really know any more. So confused. Very wow. I personally dislike having to quote my object's properties but in this case it does not bother me that much.

I guess it's rather a question of taste here. What solution would you personally use?

@jacomyal I'm sure you have an opinion about this :).
